src/main/scala/com/typesafe/conductr/sandbox/sbt/package.scala

```diff
@@ -0,0 +1,13 @@
+package com.typesafe.conductr.sandbox
+
+package object sbt {
+
+  implicit class TraversableOps[A](trav: Traversable[A]) {
+
+    def toMapWithKey[B](key: String)(f: Traversable[A] => B): Map[String, B] =
+      if (trav.isEmpty)
+        Map.empty[String, B]
+      else
+        Map(key -> f(trav))
+  }
+}
```

Comment:
This would also work:

```scala
val conductrRolesEnv = conductrRoles
  .map("CONDUCTR_ROLES" -> _)
  .groupBy   { case (key, _) => key }
  .mapValues { roles         =>  roles.map { case (key, role) => role } }
```

Scalaz has semigroups and the `|+|` which would make it very easy: http://www.nimrodstech.com/scala-map-merge/

Of course for this we don't want to introduce scalaz. The first alternative is still hard to read (IMHO). Therefor I'd stick with encapsulating this in the package.

Other ideas, WDYT?